special-interest-groups/sig-parser/member-list.md

```diff
@@ -0,0 +1,35 @@
+# SIG Parser Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2019-11-18
+
+## Active Contributors
+
+Members and their contributions:
+
+| user                                      | num_merged_PRs | earliest_PR_merged_time | latest_PR_merged_time |
+|:-----------------------------------------:|:--------------:|:-----------------------:|:---------------------:|
+| [spongedu](https://github.com/spongedu)   | 25             | 2018-11-26 12:50:20     | 2019-08-27 10:13:46   |
+| [leiysky](https://github.com/leiysky)     | 8              | 2019-07-30 11:07:34     | 2019-08-16 23:51:46   |
+| [lauhg](https://github.com/lauhg)         | 6              | 2019-08-13 12:47:51     | 2019-09-24 19:51:57   |
```

Comment:
Form the definition of `active contributors`, they should have at least 8 PRs merged within 1 year. Seems we should update this list?